gulp-tasks/utils/constants.js

```diff
@@ -1,7 +1,48 @@
+const uglify = require('rollup-plugin-uglify-es');
+const replace = require('rollup-plugin-replace');
+
 module.exports = {
   // This is a directory that should not be commited
   // to git and will be removed and rebuilt between
   // test runs.
-  BUNDLE_BUILD_DIRNAME: 'bundle-build',
-  ENVIRONMENTS: [undefined, 'production'],
+  PACKAGE_BUILD_DIRNAME: 'build',
+  BROWSER_BUILD_DIRNAME: 'browser-bundles',
+  TEST_BUNDLES_BUILD_DIRNAME: 'bundle-builds',
+
+  // This is the environments that we should use for NODE_ENV.
+  BUILD_TYPES: [undefined, 'production'],
+
+  // Every use of rollup should have minification and the replace
+  // plugin set up and used to ensure as consist set of tests
+  // as possible.
+  getDefaultRollupPlugins: (buildType) => {
```

Comment:
It's slightly odd to have this in a `constants.js` file. I think it would make more sense conceptually if you put it in with the other Rollup code.